### DISCUSSION TITLE
That’s a pass

### DISCUSSION THREAD

**Discussion by StephenHillenbrand** [MarkW06071969/Tafe202206#31] [General]
Well done everyone. The team part of the assessment has been marked as satisfactory. 
Congrats. Have a good break

**Comment by MarkW06071969** [comment]
Thanks Stephen,
That is a relief! 😊👍

Sent from Yahoo Mail on Android 
 
  On Fri, 30 Sept 2022 at 7:09 pm, ***@***.***> wrote:   


Well done everyone. The team part of the assessment has been marked as satisfactory.
Congrats. Have a good break
